parsing/dialects/intermediate/dialects/dialect1/parserImpls.ftl

```diff
@@ -3007,6 +3007,109 @@ SqlPrefixOperator PrefixRowOperator() :
     )
 }
 
+/**
+ * Parses a binary row operator like AND.
+ */
+SqlBinaryOperator BinaryRowOperator() :
+{
+    SqlBinaryOperator op;
+}
+{
+    // <IN> is handled as a special case
```

Comment:
Should we add to this comment that LIKE is also handled separately?

Reply:
Sure I'll update this